src/ApolloHooksApolloProvider.re

```diff
@@ -2,4 +2,4 @@
 external make:
```

Comment:
Does it need to have `Apollo` two times in the name? Can we have it as `ApolloProvider` as it was before?

Reply:
Yes, good one, it the unaliased name. Normally you would use `ApolloHooks.Provider`